static/locales/en-US.yaml

```diff
@@ -344,6 +344,7 @@ Settings:
     Hide Active Subscriptions: Hide Active Subscriptions
     Hide Video Description: Hide Video Description
     Hide Comments: Hide Comments
+    Hide Comment Photos: Hide Comment Photos
```

Comment:
I'm not sure about this naming. Maybe something like Hide Profile Pictures in Comments but i like to hear the other team members on this

Reply:
"Hide Comment Profile Pictures" or "Hide Comment Author Thumbnails" would be my top two suggestions.

Reply:
"Hide Profile Pictures in Comments"
"Hide Comment Profile Pictures" looks more confusing ("Comment Profile Pictures" is too much to parse

Reply:
You've changed my mind, that does sound better.